src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/custom-anomalies.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Custom anomaly detection
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+metaDescription: Learn how custom anomaly detection in New Relic notifies you of unusual app behavior. 
+---
+
+import alertsHowCustomAnomaliesWork from 'images/alerts_screenshot-full_how-custom-anomalies-work.png'
+
+import alertsAnomaliesSetUpperandLowerRanges from 'images/alerts_screenshot-full_anomalies-set-upper-and-lower-ranges.png'
+
+import alertsFacetedAnomaliesTwo from 'images/alerts_screenshot-full_faceted-anomalies-two.png'
+
+import alertsFacetedAnomalies from 'images/alerts_screenshot-full_faceted-anomalies.png'
+
+import alertsConfigureYourAnomalyThresholds from 'images/alerts_screenshot-full_configure-your-anomaly-thresholds.png'
+
+import iconAlertCritical from 'images/icon-alert-critical.png'
+
+import iconAlertWarning from 'images/icon-alert-warning.png'
+
+Custom anomalies allow your team the most flexibility when detecting unusual behavior in your system. Not only are they flexible and dynamic, custom anomalies provide your team with the capability to alert on any NQRL condition and to adjust and optimize your thresholds. Custom anomalies are built using the same advanced tuning settings as static alerting so you can ensure your team sees only incidents that are important to you. 
+
+To create a custom anomaly, first you must create a condition. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Policies > (create or select policy) > Create alert condition**: You can select APM, Browser, or NRQL condition. 
+
+Then choose anomaly as your alert condition threshold type.
+
+This threshold will predict the future behavior for the value of your data. . This prediction appears as a dotted black line on the preview chart in the anomaly condition UI. 
```

Comment:
"Will predict" to "predicts" maybe? Also there is a double period at the end of the first sentence here.

Reply:
done